bin/common.sh

```diff
@@ -24,16 +24,17 @@ function err() {
 function _rind() {
   local cmd="$1"
   shift
-  local args="$@"
   if [[ "${has_docker}" == 'true' && "${IN_DOCKER}" != '1' ]]; then
     docker run --rm \
       -v ${ENV_DIR}:${ENV_DIR} \
       -e CLOUD="$CLOUD" \
-      -e IN_DOCKER="1" \\
+      -e IN_DOCKER="1" \
```

Comment:
can we progressively (or agressively) start using single quotes for non-expandable strings?